core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -45,7 +45,7 @@
   <div class="story-details">
     <div class="story-thumbnail e2e-test-story-thumbnail"
          [ngStyle]="{background: thumbnailBgColor}">
-      <a [href]="storyLink">
+      <a [href]="storyLink" aria-label="Link to the story">
```

Comment:
Instead of 'link to the story' can it be ''link to the {{ story title }} story" ? The aria label should be more descriptive about the link it take to.

Reply:
That sounds good. Done.